### DISCUSSION TITLE
Sort by due date

### DISCUSSION THREAD

**Discussion by tasuki1978** [fixtse/o365-card#3] [Q&A]
Hello,

Is it possible to sort tasks by due date? By default it seems to sort by creation date and I haven't found any other possibilities.

Best regards

**Comment by fixtse** [comment]
Hey, sorry for the delay, let me take a look at it.
I'm working on a new version supporting the new task sensor that home assistant just introduced, so any new features would be on the beta version of the card until the new version of the integration gets released

**Reply by tasuki1978** [reply]
Thank you for your reply. I'll wait for the new version ;).
